uint8[] index          # from 0
uint8[] torque_enable  # 0 or 1
